### PR TITLE
AF-1901:restrict empty fields in Persistence section

### PR DESCRIPTION
**Pull request by ohonchar**
Added empty field validation for Properties section

### CODE REVIEW

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/persistence/PersistencePresenter.java [hunk 1]
```diff
@@ -255,4 +271,35 @@ public PropertiesListPresenter(final ManagedInstance<PropertiesItemPresenter> it
...
+        boolean isEmpty = false;
+        for (Property property : listProperty) {
+            if (property.getName() == null || property.getName().trim().isEmpty()) {
+                isEmpty = true;
+            }
+            if (property.getValue() == null || property.getValue().trim().isEmpty()) {
+                isEmpty = true;
+            }
+        }
+        boolean finalIsEmpty = isEmpty;
```
Comment (Rikkola) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/persistence/PersistencePresenter.java]: Is this assignment needed?

Comment (ohonchar) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/persistence/PersistencePresenter.java]: Thnx for comments. Done

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-backend/wildfly-multinode-kie-server.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<kie-server-state>
```
Comment (Rikkola) [inline kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-backend/wildfly-multinode-kie-server.xml]: Looks good, just this file feels out of place. Is it needed?

Comment (ohonchar) [reply kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-backend/wildfly-multinode-kie-server.xml]: Exactly, thnx

**Comment by barboras7** [comment]
jenkins execute full downstream build

**Comment by ohonchar** [comment]
@barboras7 Thank you for your comments, i've made all changes

**Comment by ohonchar** [comment]
jenkins execute full downstream build

**Comment by kie-ci3** [comment]
Build finished. 5708 tests run, 15 skipped, 0 failed.

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/test/java/org/kie/workbench/common/screens/library/client/settings/sections/persistence/PersistencePresenterTest.java [hunk 3]
```diff
@@ -307,4 +318,92 @@ private PersistenceDescriptorEditorContent newPersistenceDescriptorEditorContent
...
+        validatePropertyIsNotEmptyOnCatch();
+    }
+
+    @Test
+    public void testValidatePropertyIsNotEmptyWithEmptyValue() {
+        doNothing().when(listPresenter).addToListElement(any());
+        final List<Property> objs = Collections.singletonList(new Property("foo", "    "));
+
+        listPresenter.setup(mock(HTMLElement.class), objs, (a, b) -> {
+        });
```
Comment (barboras7) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/test/java/org/kie/workbench/common/screens/library/client/settings/sections/persistence/PersistencePresenterTest.java]: What I would do here is something like:
`private void setupProperty(String property, String value) {
doNothing().when(listPresenter).addToListElement(any());
});
final List<Property> objs = Collections.singletonList(new Property(property, value));
listPresenter.setup(mock(HTMLElement.class), objs, (a, b) -> {});
}`

and then just call `setupProperty("foo", "    ")`; 

Like this, the setup code for the methods is in one place and you only change the strings that you put there

Comment (ohonchar) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/test/java/org/kie/workbench/common/screens/library/client/settings/sections/persistence/PersistencePresenterTest.java]: Ok, got your point, thnx.

**Comment by ohonchar** [comment]
jenkins execute full downstream build

**Comment by ohonchar** [comment]
jenkins execute full downstream build

**Comment by barboras7** [comment]
jenkins retest this

**Comment by barboras7** [comment]
@ederign Could you please merge this? All checks have passed

**Pull request merged**